apps/emacs/emacs.py

```diff
@@ -12,10 +12,15 @@
 
 mod.apps.emacs = "app.name: Emacs"
 mod.apps.emacs = "app.name: emacs"
+mod.apps.emacs = "app.name: /GNU Emacs/"
```

Comment:
Why the slashes? How would this differ from `app.name: "GNU Emacs"`?

Reply:
I had asked about this on Slack, b/c I'm definitely no expert on how the context matchers work so defer to someone else on how to do this... but slashes make it match anything with "GNU Emacs" in the name. I also considered just changing `app.name: Emacs` to `app.name: /Emacs/`

actually, no changes to `app.name` are needed if we have the `emacs.exe` matcher and maybe this line should be omitted. here is the output from `copy context` with Emacs on Windows 10 for me:

```
os: windows
and app.name: GNU Emacs: The extensible self-documenting text editor
os: windows
and app.exe: emacs.exe
```

Reply:
Thanks! I think this is probably fine, especially since relying exclusively on .exe names on Windows seems a bit flaky to me.

Reply:
Matching it anywhere could be a bit overboard, perhaps the regex should be anchored instead? i.e.
```
app.name: /^GNU Emacs/
```